### ISSUE TITLE
Add service mesh version support to sm.proto

### ISSUE THREAD

**Issue by leecalcote** [layer5io/service-mesh-performance-specification#71]
**Current Behavior**
[sm.proto](https://github.com/layer5io/service-mesh-performance-specification/blob/master/sm.proto) identifies service mesh by name, but excludes other metadata like service mesh version number.

**Desired Behavior**
SPMS will need to afford for identifying a given service mesh's version #.

**Comment by leecalcote** [comment]
// @kumarabd

**Comment by kumarabd** [comment]
@leecalcote Will have a discussion with @kanishkarj and take this ahead.

**Comment by kushthedude** [comment]
There is already a field present for the same `string mesh-build version`

**Comment by kumarabd** [comment]
Yes, we need to clean up the proto a little. Looking into that. @kushthedude

**Comment by matrixkavi** [comment]
A lot of changes have gone into the code since the issue was filed. We do not have sm.proto in repo but have a folder with 3 proto files here: https://github.com/service-mesh-performance/service-mesh-performance/tree/6ecbce7985dddb5577509f25a1ab67959da8ca68/protos . Do we still need to do anything for this issue?

**Comment by matrixkavi** [comment]
cc: @Rajesh-Nagarajan-11 - I assume we can close this one?
